app/views/application/_navbar.html.erb

```diff
@@ -1,9 +1,36 @@
-<nav>
-<h3><%= link_to t("nav.home"), root_path %> |
-<% if current_user %>
-  <%= link_to t("nav.signout"), session_path(current_user), method: :delete %> |
-<% else %>
-  <%= link_to t("nav.signup"), new_users_path %> |
-  <%= link_to t("nav.signin"), new_session_path %></h3>
-<% end %>
-</nav>
+<div class="navbar">
+  <% if current_user %>
+    <% if current_user.admin? %>
+      <div class="dropdown">
+        <button class="dropbtn"><%= t("nav.admin") %></button>
+        <div class="dropdown-content">
+          <a><%= link_to t("nav.user_index"), admin_users_path %></a>
```

Comment:
Actually, the `<a>` html tag is not needed because it will be generated by the `link_to` helper method

Reply:
All redone - dropdown menu restyled in Done in ad-css-styling sha: b2d6f90